packages/gcf-utils/test/gcf-utils.ts

```diff
@@ -19,6 +19,8 @@ import {Options} from 'probot';
 import * as express from 'express';
 import sinon from 'sinon';
 import nock from 'nock';
+import {expect} from 'chai';
```

Comment:
For the sake of consistency, could we avoid `chai` and use `assert`?

Reply:
Done. 

There are a number of bots that need to conform to this as well. I'll make a separate issue to enforce consistency.

* auto-label
* buildcop
* generate-bot
* publish
* snippet-bot